go/signedexchange/README.md

```diff
@@ -46,8 +46,8 @@ Here, we assume that you have an access to an HTTPS server capable of serving st
 
 1. Convert the PEM certificate to `application/cert-chain+cbor` format using `gen-certurl` tool.
     ```
-    # Fill in dummy data for OCSP/SCT, since the certificate is self-signed.
-    gen-certurl -pem cert.pem -ocsp <(echo ocsp) -sct <(echo sct) > cert.cbor
+    # Fill in dummy data for OCSP, since the certificate is self-signed.
+    gen-certurl -pem cert.pem -ocsp <(echo ocsp) > cert.cbor
```

Comment:
I removed `-sct` here since SCT is not necessary for self-signed certs.

I plan to add a separate section for instructions to create signed exchanges with trusted certificates, and will explain the usage of `-sctDir` there.

Reply:
sgtm